action.yml

```diff
@@ -1,48 +1,49 @@
 name: terraform-docs
-author: Derek Rada
-description: A Github action for generating terraform module documentation using terraform-docs and gomplate.
+author: terraform-docs Authors
+description: A Github action for generating Terraform module documentation using terraform-docs and gomplate.
+
 inputs:
-  tf_docs_working_dir:
+  working_dir:
```

Comment:
As you can see I've removed the already implied `tf_docs_` prefix from inputs and also I've been thinking to rename them to hyphen-separated rather than snake case. What do you think of this?

Reply:
I am perfectly fine with removing the tf_docs_ prefix but if you do hyphen github won't be able to pass them down as variables as I am fairly certain `-` will not be a valid variable name but I could be wrong.

Reply:
To be specific about the variables, they get passed in as environment variables and bash doesn't support `-`.

Reply:
I see, thanks. I've actually found a way to pass `-` separated inputs down to the docker image without losing them. I'm gonna push something soon for it.
I personally prefer hyphens because they are more readable.